django/template/engine.py

```diff
@@ -203,6 +203,15 @@ def render_to_string(self, template_name, context=None,
                          context_instance=_context_instance_undefined,
                          dirs=_dirs_undefined,
                          dictionary=_dictionary_undefined):
+        return ''.join(self._stream(template_name, context, context_instance, dirs, dictionary))
+
+    def stream(self, template_name, context=None):
+        return self._stream(template_name, context=context)
+
+    def _stream(
+            self, template_name, context=None,
+            context_instance=_context_instance_undefined,
+            dirs=_dirs_undefined, dictionary=_dictionary_undefined):
```

Comment:
Is the distinction between ``stream`` and ``_stream`` necessary here? Could we just define ``stream``. It would take the deprecated arguments, but we wouldn't need to document that.